### ISSUE TITLE
Adjust frame rates on slow devices

### ISSUE THREAD

**Issue by etchenko** [arkis/arkis.io#10]
On slower and older computers the mist effect lags and ends up slowing down the entire browser. There should be an option for older computers to be able to not have the animations.

**Comment by controversial** [comment]
Yeah. JavaScript [doesn't currently provide easy ways to detect less powerful computers](http://stackoverflow.com/q/15921481/4414003). I think maybe the best way to work around this would be to go for a lower frame rate on slow devices.

The default target frame rate is 25 frames per second. The animation tries to advance at this speed, and slows down if it can't. This also causes a lot of slowness though.

If the simulation started with target 25 FPS, it could measure what the actual frame rates were over the first 5-ish seconds, then it could adjust the simulation to match that. So if a client computer was only hitting 15 frames per second, the simulation could measure this and adjust the target to 15 frames per second.

For really slow computers (less than 10FPS) the simulation could just be stopped and left as a static rendered thing
